src/app/layout.module.scss

```
@@ -0,0 +1,11 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
```

```suggestion
  @include flex-column(0rem);
```

요곤 어떤지,, 하는 그냥 작은 저의 생각